### DISCUSSION TITLE
Dashboard support

### DISCUSSION THREAD

**Discussion by sushant-pradhan** [hyperjumptech/monika#372] [Ideas]
Is there a plan to adding dashboard support similar to what Uptime robot provides:

![uptime-robot-dashboard](https://user-images.githubusercontent.com/16751240/132143522-3e7a11c3-cf11-4249-99c9-04e7b6304126.jpg)

**Comment by sapiderman** [comment]
Hi thanks for the feedback! Monika recently supported the use of prometheus. There's a 2 article write up on it, take a look here: https://medium.com/hyperjump-tech/collecting-monika-with-prometheus-9faa7d484a30

Let us know what you think,

**Reply by bengoldenberg** [reply]
i will be happy if you will make a complete dashboard that we will be able to import and use for our own environments

**Reply by sapiderman** [reply]
@bengoldenberg we'll see what we can do. Actually not a bad idea.

**Reply by sapiderman** [reply]
> i will be happy if you will make a complete dashboard that we will be able to import and use for our own environments

@bengoldenberg  Can you give more detaisl on your environment? your use case? Then we can make sure the sample dashboard works.

**Reply by bengoldenberg** [reply]
> @bengoldenberg we'll see what we can do. Actually not a bad idea.

just a generic dashboard with all the metrics, we will adjust this for our needs.

**Reply by amitde69** [reply]
From our standpoint, if monika produces prom metrics then it is a no-brainer to also provide a basic grafana dashboard to present those metrics on.
It will be easier for newcomers to adopt it that way.
